bin/init

```diff
@@ -62,4 +61,4 @@ setup_database() {
 
 setup_database
 bundle exec rake db:migrate
-bundle exec "rails s -b 0.0.0.0 -p $VELUM_PORT --pid /tmp/puma.pid Puma"
```

Comment:
Binding on all interfaces is generally not recommended (0.0.0.0), is there a way to explicitly bind to the public interface?